util/gen-week.js

```diff
@@ -93,7 +93,7 @@ function createMatches(params) {
 }
```

Comment:
Not sure if we really need gen-week anymore. spawn-matches is now used to auto-gen the matches each week. However, if we want to create a specific week, this would still be easier. I'll keep it around until spawn-matches can accept params to specify a week.